terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/main.tf

```diff
@@ -80,22 +80,23 @@ data "aws_route53_zone" "cloud_platform" {
 ##########
 
 locals {
+  # prod_workspace refer to all production workspaces which have active monitoring set and followed
   prod_workspace = {
     manager = true
     live    = true
+    live-2  = true
     default = false
   }
 
-  cloudwatch_workspace = {
-    manager = false
+  # live_workspace refer to all production workspaces which have users workload in it
+  live_workspace = {
     live    = true
     live-1  = true
     default = false
   }
```

Comment:
So we need to add live-2 here in live_workspace?

Reply:
Thanks Sablu. Thats correct. Have added.